x-pack/plugins/security_solution/public/common/components/threat_match/helpers.tsx

```diff
@@ -22,14 +22,16 @@ import { Entry, FormattedEntry, ThreatMapEntries, EmptyEntry } from './types';
  */
 export const getFormattedEntry = (
   indexPattern: IndexPattern,
+  threatIndexPatterns: IndexPattern,
   item: Entry,
   itemIndex: number
 ): FormattedEntry => {
   const { fields } = indexPattern;
+  const { fields: threatFields } = threatIndexPatterns;
   const field = item.field;
   const threatField = item.value;
   const [foundField] = fields.filter(({ name }) => field != null && field === name);
```

Comment:
Could these be memoized? My only hesitance with memoizing would be that it may just run on every render regardless because the hook doesn't do a deep equal to my understanding.

Reply:
It is above in the React Component which I think is what we want

```ts
    const entries = useMemo(
      (): FormattedEntry[] =>
        indexPattern != null && listItem.entries.length > 0
          ? getFormattedEntries(indexPattern, threatIndexPatterns, listItem.entries)
          : [],
      [listItem.entries, indexPattern, threatIndexPatterns]
    );
```